_analyzers/token-filters/asciifolding.md

```diff
@@ -0,0 +1,135 @@
+---
+layout: default
+title: ASCIIFolding
+parent: Token filters
+nav_order: 20
+---
+
+# ASCIIFolding token filter
+
+`asciifolding` is a token filter that converts non-ASCII characters into their closest ASCII equivalents. For example *é* becomes *e*, *ü* becomes *u* and *ñ* becomes *n*. This process is also known as *transliteration*.
+
+
+`asciifolding` token filter offers a number of benefits:
+
+  - __Enhanced Search Flexibility__: Users often omit accents or special characters when typing queries. ASCIIFolding ensures that such queries still return relevant results.
+  - __Normalization__: Standardizes the indexing process by ensuring that accented characters are consistently converted to their ASCII equivalents.
+  - __Internationalization__: Particularly useful for applications dealing with multiple languages and character sets.
+
+*Loss of Information*: While ASCIIFolding can simplify searches, it might also lead to loss of specific information, particularly if the distinction between accented and non-accented characters is significant in the dataset.
```

Comment:
```suggestion
While the `asciifolding` token filter can simplify searches, it might also lead to loss of specific information, particularly if the distinction between accented and non-accented characters is significant in the dataset.
```